### ISSUE TITLE
Multi-line strings do not seem to be exported correctly to CSV

### ISSUE THREAD

**Issue by Fs00** [activeadmin/activeadmin#6907]
I observed that new-line characters in string fields are not escaped when exporting index page results to a CSV file, causing broken CSVs in case of multiline string fields.

### Expected behavior

Given a record that has a string field with the following content:
```
string that
spans
multiple lines
```

That string should be written in the CSV as follows:

```
[...],string that\nspans\nmultiple lines,[...]
```

### Actual behavior

The example string above gets written in the CSV in the following way:
```
[...],string that
spans
multiple lines,[...]
```
thus introducing broken entries in the CSV file.

### How to reproduce

- Create a new entity and fill one (or more) of its text fields with a multiline string
- Go back to the index page
- Export the index results in CSV format
- Observe the issue in the generated CSV

**Comment by deivid-rodriguez** [comment]
I did a quick test and I got what I would have expected in this case. A quoted string with actual line breaks inside. This works just fine with my spreadsheet application, and matches [the standard](https://tools.ietf.org/html/rfc4180#section-3):

>    6.  Fields containing line breaks (CRLF), double quotes, and commas should be enclosed in double-quotes.  For example:
>
>       "aaa","b CRLF
>       bb","ccc" CRLF
>       zzz,yyy,xxx

From your issue it's unclear whether the field with line breaks is getting quoted. That sounds like an issue and we'd need to investigate why it's happening in your case since simple tests suggest the correct behaviour.

**Comment by Fs00** [comment]
Hello @deivid-rodriguez, thanks for your reply.
I can confirm that the multi-line field gets quoted, but CRLF characters do not get escaped.
Setting `ActiveAdmin.application.csv_options[:force_quotes] = true` didn't have an impact on the issue.

**Comment by Fs00** [comment]
After a bit more of analysis, I've noticed that even though the original string had CRLF in it, in the CSV it was written with LF only.
That didn't change how Excel parsed the line though 🤔

**Comment by deivid-rodriguez** [comment]
mmmm :thinking:. I duckduckgo'ed "excel CSV line breaks" and got this as a first result: https://www.excelchamp.net/adding-line-breaks-in-a-csv-file/, which suggests that Excel should deal with this just fine. We must be missing something.

**Comment by Fs00** [comment]
I tried importing the CSV file created by AA in LibreOffice Calc and multi-line string fields were parsed correctly.
This is definitely an Excel-side issue. I have seen other people having this kind of problem with that program (see https://stackoverflow.com/questions/2668678/importing-csv-with-line-breaks-in-excel-2007).

Thank you for your replies! At least I learned that CSV with line breaks in quoted fields are legal :)
